notebooks/demo/accelerating_networkx.ipynb

```diff
@@ -0,0 +1,614 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "xkg10FrNThrK"
+      },
+      "source": [
+        "# Let's Get the Environment Setup\n",
+        "This notebook will demonstrate NetworkX both with and without GPU acceleration provided by the `nx-cugraph` backend.\n",
+        "\n",
+        "`nx-cugraph` is available as a package installable using `pip`, `conda`, and [from source](https://github.com/rapidsai/nx-cugraph).  Before importing `networkx`, lets install `nx-cugraph` so it can be registered as an available backend by NetworkX when needed.  We'll use `pip` to install.\n",
+        "\n",
+        "NOTES:\n",
+        "* `nx-cugraph` requires a compatible NVIDIA GPU, NVIDIA CUDA and associated drivers, and a supported OS. Details about these and other installation prerequisites can be seen [here](https://docs.rapids.ai/install#system-req).\n",
+        "* The `nx-cugraph` package is currently hosted by NVIDIA and therefore the `--extra-index-url` option must be used.\n",
+        "* `nx-cugraph` is supported on specific 11.x and 12.x CUDA versions, and the major version number must be known in order to install the correct build (this is determined automatically when using `conda`).\n",
+        "\n",
+        "To find the CUDA major version on your system, run the following command:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "NMFwzc1I95BS"
+      },
+      "outputs": [],
+      "source": [
+        "!nvcc --version"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "i91Yj-yZ-nGS"
+      },
+      "source": [
+        "From the above output we can see we're using CUDA 12.x so we'll be installing `nx-cugraph-cu12`. If we were using CUDA 11.x, the package name would be `nx-cugraph-cu11`. We'll also be adding `https://pypi.nvidia.com` as an `--extra-index-url`:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "mYYN9EpnWphu"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install nx-cugraph-cu12 --extra-index-url=https://pypi.nvidia.com"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0h1K-7tI_AZH"
+      },
+      "source": [
+        "Of course, we'll also be using `networkx`, which is already provided in the Colab environment. This notebook will be using features added in version 3.3, so we'll import it here to verify we have a compatible version."
```

Comment:
You could mention that the feature added inversion 3.3 is the networkx.config package which is referenced here.
https://networkx.org/documentation/stable/reference/backends.html#module-networkx.utils.configs

Reply:
In the interest of getting these merged, I think it would be best to add these in a future PR. I'll make note of these, thak you.